### ISSUE TITLE
Publish: Warning for ccbs not saved/force save

### ISSUE THREAD

**Issue by Tw0bit** [cocos2d/CocosBuilder#114]
When publishing the ccb to ccbi, there is no warning about unsaved changes. Would it be possible to have a prompt/warning about changes not being saved or an option to force saving prior to publish?

**Comment by vlidholt** [comment]
Good idea. Not sure it will make it to the release of 2.1, but definitely in future versions!
